frontend/src/routes/LandingPage/index.jsx

```diff
@@ -170,6 +177,7 @@ const Notice = styled.p`
   font-style: italic;
   font-size: 1rem;
   max-width: 600px;
+  width: 600px;
```

Comment:
Pro-tip for later is to set the default width to `100%`, instead of hard coding the same value used somewhere else. I'll let it slide though, since we'll revert this :p